### ISSUE TITLE
Minor cosmetic issues in explainer example code

### ISSUE THREAD

**Issue by fordacious** [immersive-web/webxr-hand-input#62]
"Great example code in the explainer--relatively easy to follow. I noticed a potential issue in the second code example for Hand Interaction--`checkInteraction` runs once each frame, and thus when the hand enters the radius of the button in 3D space, the button will very rapidly toggle between `pressed` states 🙂. Not sure it matters from a pedagogical example though."

**Comment by Manishearth** [comment]
Yeah, perhaps we could add some debouncing code, or maybe a "`// handle debounce here` comment
